docs/operators/jdbcPollInputOperator.md

```diff
@@ -0,0 +1,155 @@
+    <value>4</value>
+  </property>
+```
+
+where {OperatorName} is the name of the JDBC Poller operator.
+
+This will create 5 operator instances in all. Four of these will fetch the data which is currently in the table. We call these static non-polling partitions. An additional partition will be created which will read any newly added data. We call such a partition as a polling partition, as it "polls" for newly added data. Changing the value of the property will create corresponding number of static partitions.
+
+#### Dynamic Partitioning
+Not supported.
+
+## Operator Information
+1. Operator location: ***malhar-library***
+2. Available since: ***3.5.0***
+3. Operator state: ***Evolving***
+4. Java Packages:
+    * Operator: ***[com.datatorrent.lib.db.jdbc.AbstractJdbcPollInputOperator](https://www.datatorrent.com/docs/apidocs/com/datatorrent/lib/db/jdbc/AbstractJdbcPollInputOperator.html)***
+
+JDBC Poller is **idempotent**, **fault-tolerant** and **statically partitionable**.
+
+## AbstractJdbcPollInputOperator
+This is the abstract implementation that serves as base class for polling messages from jdbc store. It can be extended to modify functionality or add new capabilities. This class doesn’t have any ports, so concrete subclasses will need to provide them if necessary.
+
+![AbstractJdbcPollInputOperator.png](images/jdbcinput/operatorsClassDiagram.png)
+
+###<a name="AbstractJdbcPollInputOperatorProps"></a>Properties of AbstractJdbcPollInputOperator
+Several properties are available to configure the behavior of this operator and they are summarized in the table below. Of those only store, tableName, columnsExpression and key are mandatory ones. Those properties can be set like this:
+
+```xml
+<property>
+  <name>dt.operator.{OperatorName}.prop.tableName</name>
+  <value>mytable</value>
+</property>
+<property>
+  <name>dt.operator.{OperatorName}.prop.columnsExpression</name>
+  <value>column1,column2,column4</value>
+</property>
+<property>
+  <name>dt.operator.{OperatorName}.prop.key</name>
+  <value>keycolumn</value>
+</property>
+<property>
+  <name>dt.operator.{OperatorName}.prop.store.databaseDriver</name>
+  <value>com.mysql.jdbc.Driver</value>
+</property>
+<property>
+  <name>dt.operator.{OperatorName}.prop.store.databaseUrl</name>
+  <value>jdbc:mysql://localhost:3306/mydb</value>
+</property>
+<property>
+  <name>dt.operator.{OperatorName}.prop.store.connectionProps</name>
+  <value>user:myuser,password:mypassword</value>
+</property>
+```
+
+* If you need to filter the table records, set `whereCondition` which will be added to the generated sql query.
+* If you have huge database to scan set `partitionsCount` to a higher number to increase read parallelism.
+* The operator uses PreparedStatement, a precompiled SQL statement to fetch records from database table. You can set `fetchSize` to hint database driver to restrict number of rows to fetch in one call. The remaining rows of resultset will be fetched in subsequent calls.
```

Comment:
"to hint" => "as a hint to the"
Add explicit note that some DB drivers may not honor the hint.